apps/ledger-live-mobile/src/index.js

```diff
@@ -420,33 +437,16 @@ const DeepLinkingNavigator = ({ children }: { children: React$Node }) => {
       wcContext.initDone,
       wcContext.session.session,
       filteredManifests,
+      liveAppProviderInitialized,
     ],
   );
 
   const [isReady, setIsReady] = React.useState(false);
 
   useEffect(() => {
-    if (!wcContext.initDone) {
-      return;
-    }
-    if (
```

Comment:
this was totally broken (i implemented it) so it was always false and anyway dangerous logic that blocks the rendering of the app in case the removeLiveAppState somehow breaks